prac_05/emails.py

```diff
@@ -0,0 +1,24 @@
+def main():
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = strip_name_from_email(email)
+        confirm = input(f"Is you name {name}? (Y/n)")
```

Comment:
I am just wondering if there is suppose to be no space after the text input near the bracketed yes or no?

Reply:
Looking at the sample output there is meant to be one, added it now